scripts/check-release-config.mjs

```
@@ -0,0 +1,147 @@
+  return {
+    hash: 'a1b2c3d4e5f60718293a4b5c6d7e8f9012345678',
+    subject: message.split('\n')[0],
+    message,
+  };
+}
+
+function context(commits) {
+  return {
+    commits,
+    logger: silentLogger,
+    cwd: process.cwd(),
```

ESLint

[error] 'process' is not defined.

(no-undef)